### ISSUE TITLE
odo service create followed by odo service delete fails

### ISSUE THREAD

**Issue by jeffmaury** [openshift/odo#1368]
[kind/bug]

<!--

Welcome! - We kindly ask you to:

  1. Fill out the issue template below 
  2. Use the chat and talk to us if you have a question rather than a bug or feature request.

The chat room is at: https://chat.openshift.io/developers/channels/odo

Thanks for understanding, and for contributing to the project!

-->

## What versions of software are you using?
- Operating System: Win10 Pro
- Output of `odo version`:

```
odo v0.0.19 (61f278a1)

Server: https://172.17.188.37:8443
OpenShift: v3.10.45
Kubernetes: v1.10.0+b81c8f8
```

## How did you run odo exactly?

`odo service create postgresql-persistent --plan default s1 && odo service delete s1 -f`

## Actual behavior

```
 ✓  Service 's1' was created                                                                                                                                                                   
Progress of the provisioning will not be reported and might take a long time.                                                                                                                  
You can see the current status by executing 'odo service list'                                                                                                                                 
                                                                                                                                                                                               
 ✗  unable to delete service because Service Catalog is not enabled in your cluster:                                                                                                           
unable to get the service list: no condition in status for {TypeMeta:{Kind: APIVersion:} ObjectMeta:{Name:s1 GenerateName: Namespace:myproject SelfLink:/apis/servicecatalog.k8s.io/v1beta1/nam
espaces/myproject/serviceinstances/s1 UID:b86386ee-392c-11e9-ab64-0242ac120006 ResourceVersion:686 Generation:1 CreationTimestamp:2019-02-25 19:39:44 +0100 CET DeletionTimestamp:<nil> Deletio
nGracePeriodSeconds:<nil> Labels:map[app:0-0-19-oqjj app.kubernetes.io/component-name:s1 app.kubernetes.io/component-type:postgresql-persistent app.kubernetes.io/name:0-0-19-oqjj] Annotations
:map[] OwnerReferences:[] Initializers:nil Finalizers:[kubernetes-incubator/service-catalog] ClusterName:} Spec:{ClusterServiceClassExternalName:"postgresql-persistent", ClusterServicePlanExt
ernalName:"default"} Status:{Conditions:[] AsyncOpInProgress:false OrphanMitigationInProgress:false LastOperation:<nil> DashboardURL:<nil> CurrentOperation: ReconciledGeneration:0 ObservedGen
eration:0 OperationStartTime:<nil> InProgressProperties:<nil> ExternalProperties:<nil> ProvisionStatus: DeprovisionStatus:NotRequired}}                                                        
```

## Expected behavior
Either success or refusal but for another reason that Service Catalog is not enabled in your cluster

## Any logs, error output, etc?

**Comment by metacosm** [comment]
@jeffmaury you need to wait for the service to be created first using the `-w` flag. That said, I agree that the error message you get is incorrect and/or confusing.

**Comment by metacosm** [comment]
That said, I haven't been able to reproduce the issue locally.

**Comment by jeffmaury** [comment]
Can't find the `-w` flag

**Comment by kadel** [comment]
> Can't find the `-w` flag

This flag has been added recently, and it is not in the latest  v0.0.19, but it will be there in  v0.0.20 that will be released this week

**Comment by metacosm** [comment]
> Can't find the `-w` flag

which version of `odo` are you using?

**Comment by metacosm** [comment]
@jeffmaury can you give #1374 a try, please?

**Comment by kadel** [comment]
fixed in  #1374
